assets/charts/components/istio-operator/values.yaml

```diff
@@ -1,8 +1,29 @@
 hub: docker.io/istio
-tag: 1.6.3
+tag: 1.7.3
```

Comment:
hub and tag were manually modified because istio is still shipping bad config in releases.